typings/index.d.ts

```diff
@@ -1374,8 +1374,7 @@ declare module 'discord.js' {
     public readonly ids: number[];
     public mode: ShardingManagerMode;
     public parentPort: any | null;
-    public broadcastEval(script: string): Promise<any[]>;
-    public broadcastEval<T>(fn: (client: Client) => T): Promise<T[]>;
+    public broadcastEval<T>(fn: (client: Client) => T | string): Promise<T[]>;
```

Comment:
`(client: Client) => T | string` implies a function that returns either `T` or `string`, it's not the same as doing `((client: Client) => T) | string`.

Same for the line 1406.